ledger/catchpointwriter_test.go

```diff
@@ -343,13 +327,16 @@ func TestFullCatchpointWriter(t *testing.T) {
 	}
 	err = repackCatchpoint(
 		context.Background(), catchpointFileHeader, biggestChunkLen,
-		catchpointDataFilePath, catchpointFilePath)
+		datapath, filepath)
 	require.NoError(t, err)
 
 	// create a ledger.
```

Comment:
@jannotti If I've followed the logic accurately, then I think there's test duplication that can be removed via https://github.com/algorand/go-algorand/pull/4582.  Can you have a look to see if you agree?

Reply:
change will appear in code-review PR